application/paths.py

```
@@ -2,10 +2,22 @@
+import sys
 from pathlib import Path
 
+# ── frozen 模式检测（PyInstaller / Tauri bundle） ──────────────────────────
+_FROZEN = getattr(sys, "frozen", False)
+_MEIPASS = getattr(sys, "_MEIPASS", None) if _FROZEN else None
+
 # application/paths.py → 仓库根目录 aitext/
-AITEXT_ROOT = Path(__file__).resolve().parents[1]
+# 冻结时 __file__ 在 sys._MEIPASS 内，parent/parent 指向 bundle 根
+if _FROZEN and _MEIPASS:
+    # _MEIPASS = bundle/Resources/（mac）或 bundle/（windows）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)